### PR TITLE
German signup

### PR DESCRIPTION
**Pull request by isaacseymour**
@jutta-frieden could you have a look at the translations?

@samjewell could you review the codes?

### CODE REVIEW

File: app/pages/pricing/pricing.de.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default class PricingDe extends React.Component {
                     </IfLinkExists>
                     <IfLocale hasInstantSignup>
                       <li className='pricing-options__list-button'>
-                        <a href='/merchants/new' className='btn u-size-full'>Sign up today</a>
+                        <a href='/merchants/new' className='btn u-size-full'>TRANSLATE ME! Sign up today</a>
```
Comment (Ken-Lo) [inline app/pages/pricing/pricing.de.js]: Don't forget to update this to `<Href to='signup.path' className='btn u-size-full'>` :-)

File: app/messages/fr.js [hunk 2]
```diff
@@ -279,9 +279,9 @@ export default {
     nav_title: 'Soutien',
   },
   signin: {
-    path: 'http://manage.gocardless.com',
+    path: 'https://manage.gocardless.com',
```
Comment (samjewell) [inline app/messages/fr.js]: good fix, thanks for that!

**Comment by samjewell** [comment]
Hey man, great work so far. 
on `/de-de/preise` the JETZT ANMELDEN button for basic still points at old dashboard

**Comment by samjewell** [comment]
I'd also add a signup link to the sticky nav as well:
screen shot 2015-07-03 at 09 52 41

**Comment by samjewell** [comment]
Hey- a bit outside scope, but could you point the sandbox-signup at the new signup URL?
On /Pro
screen shot 2015-07-03 at 09 55 49
should now be pointing to https://manage-sandbox.gocardless.com/signup

**Comment by samjewell** [comment]
I suppose I'd also add the signup CTA to the bottom of the homepage, as we've done on the french homepage... if you fancy it?
screen shot 2015-07-03 at 10 00 54

**Comment by isaacseymour** [comment]
@Ken-Lo already pointed out the link. 

Fixing all the broken sandbox signup links is definitely one for a separate PR. I'll fix the German ones here, and French/English ones elsewhere.

**Comment by samjewell** [comment]
:+1: with comments - looking great man!

**Comment by isaacseymour** [comment]
:de: :de: :de: :de: :de:

**Pull request merged**